### ISSUE TITLE
Declaring a property class doesn't seem to work

### ISSUE THREAD

**Issue by mthuurne** [PyCQA/pylint#2365]
### Steps to reproduce

Put this in pylintrc:
```
[BASIC]
property-classes=Doubler
```

Then run pylint on the following code:
```python
class Doubler:

    def __init__(self, method):
        self.__method = method

    def __get__(self, instance, owner):
        if instance is None:
            return self
        return self.__method(instance) * 2

    def __set__(self, instance, value):
        raise AttributeError('read-only property')

    def __delete__(self, instance):
        raise AttributeError('read-only property')

class Clazz:

    @Doubler
    def number(self):
        return 123

print(Clazz().number.bit_length())
```

### Current behavior

The following message is issued:
```
23:6: E1101: Method 'number' has no 'bit_length' member (no-member)
```

### Expected behavior

Ideally, no message would be issued. However, even if pylint got the type inference wrong, it should at least say `Attribute 'number'` instead of `Method 'number'`.

### pylint --version output

```
pylint 2.0.1
astroid 2.0.1
Python 3.4.6 (default, Mar 22 2017, 12:26:13) [GCC]
```

**Comment by bgiard** [comment]
For people researching out to do what @mthuurne was trying to achieve, one can add to pylintrc:

```
[MASTER]
init-hook="import astroid.bases; astroid.bases.POSSIBLE_PROPERTIES.add('custom_property')"
```

Assuming the name of the decorator is ```custom_property```.
